example/producer.js

```diff
@@ -58,18 +61,22 @@ const logger = console;
   // create first Task
   const firstTask = steveo.task('test-topic', () => {}, attributes);
   const secondTask = steveo.task('test-spam', () => {}, attributes);
-  await steveo.runner().createQueues();
+  if(process.env.ENGINE !== 'kafka') {
+    await steveo.runner().createQueues();
```

Comment:
@arshu16 any reason from an API perspective that `steveo.runner().createQueues();` can't call `steveo.adminClient().createTopic()`

Also from a meta data perspective and API consumer the parameters `num_partitions: 2` and  `replication_factor: 1` should prob be stored on the task in the registry

`steveo.task('name', callback, opts={})`

Reply:
where opts = { num_partitions: 2 and replication_factor: 1 }

Reply:
Removed some extra options on `steveo.task`, added attributes as an object, and create queues in the kafka runner.